src/bin/download_sysext.rs

```diff
@@ -376,73 +83,27 @@ fn main() -> Result<(), Box<dyn Error>> {
                 Some(io::read_to_string(file)?)
             }
         }
-        None => None,
+        None => {
+            return Err("failed to get input xml file".into());
+        }
```

Comment:
This change looks weird. Does this mean that `--input-xml` always needs to be passed?